nautobot/core/api/urls.py

```diff
@@ -53,7 +53,7 @@
     path("graphql/", GraphQLDRFAPIView.as_view(), name="graphql-api"),
     # Plugins
     path("plugins/", include((plugin_api_patterns, "plugins-api"))),
-    # Core Apps
+    # Core Apps - TODO, this should probably be moved into `core_api_patterns`, if we even find we need it...
```

Comment:
Or possibly these all go under `/api/ui/...` instead?

Reply:
https://github.com/nautobot/nautobot/issues/3240 👀

Reply:
Added a link to #3240 in this comment.